components/ILIAS/UI/src/Component/Layout/Page/Factory.php

```diff
@@ -109,4 +110,39 @@ public function standard(
         string $short_title = '',
         string $view_title = ''
     ): Standard;
+
+    /**
+     * ---
+     * description:
+     *   purpose: The Mail Page is used to render HTML content for mailing purposes.
+     *   composition: >
+     *    The Mail Page consists of HTML content, a header and a footer.
+     *    The header contains the logo and the installation-text.
+     *    The footer contains the installation-text and a link to ILIAS.
+     *    The stylesheet path is referees to the CSS file that is used to style the page and will be included in the HTML.
+     *
+     * rules:
+     *   usage:
+     *     1: The Mail Page MUST be rendered with content.
+     *     2: The Mail Page MUST be rendered with a logo.
```

Comment:
The logo is optional AFAIK.

```suggestion
     *     2: The Mail Page SHOULD be rendered with a logo.
```